htdocs/comm/propal.php

```diff
@@ -293,7 +293,8 @@
 					$object->note = GETPOST('note');
 					$object->statut = 0;
 
-					$id = $object->create_from($user);
+					// the create is done below and further more the existing create_from function is quite hilarating
+					//$id = $object->create_from($user);
```

Comment:
When creating a proposal from another, we must alos copy lines, where are setting of lines if we remove this ?